frame_metadata/_downloaded/mod.ts

```diff
@@ -0,0 +1,29 @@
+// This file was generated by `_tasks/download_frame_metadata.ts`
```

Comment:
No reason for this file to be generated; it's only ever consumed as a full object, so `knownClients` can just be mapped over at runtime.

Reply:
We previously used a `setup` fn from `test-common.ts` to avoid reloading metadata between tests, so either way, we have an extra file. Felt fitting to place that file alongside the SCALE files with which it is concerned (and to make sure that––no matter how the download task URL list is defined, the mod file is generated to reflect all downloaded files). Any concerns?

Reply:
Fair enough.